code/src/reducers/todos.js

```diff
@@ -0,0 +1,57 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  list: {
+    name: 'todos',
+    items: [],
+    options: [
+      { value: 'school', label: 'School' },
+      { value: 'work', label: 'Work' },
+      { value: 'personal', label: 'Personal' },
+      { value: 'home', label: 'Home' },
+      { value: 'other', label: 'Other' },
+    ],
+    selectedStatusFilter: '',
+  },
+};
+
+export const todos = createSlice({
+  name: 'todos',
+  initialState: initialState,
+  reducers: {
+    addTodo: (state, action) => {
+      const { todoItemInfo } = action.payload;
+      state.list.items.push(todoItemInfo);
+    },
+
+    setComplete: (state, action) => {
```

Comment:
This action is not only settting complete, it's setting incomplete too, right? ;)